### PR TITLE
feat: My Collection form artist avatar

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses [CX-3216]

## Description

Introduces the artist avatar in the My Collection form. The artist avatar will show up if the user has selected an artist in the flow or if the user edits an artwork.

With this change, it's no longer possible to change the artist of an existing artwork in My Collection.

Screenshot 2022-12-23 at 14 08 03

[CX-3216]: https://artsyproduct.atlassian.net/browse/CX-3216?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/Components/ArtistAutocomplete.tsx [hunk 1]
```diff
@@ -29,18 +29,18 @@ type SubmissionImage =
...
 interface ArtistAutocompleteOption extends AutocompleteInputOptionType {
-  image: SubmissionImage
+  option: AutocompleteArtist
 }
 
 export const ArtistAutoComplete: React.FC<{
   onError: () => void
   onChange?: (value: string) => void
-  onSelect: ({ artistId, artistName }) => void
+  onSelect: (artist: AutocompleteArtist | null) => void
```
Comment (olerichter00) [inline src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/Components/ArtistAutocomplete.tsx:43]: Returning the artist here to be able to show the artist's avatar.

File: src/Apps/MyCollection/Routes/EditArtwork/Components/ArtistAvatar.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { Avatar, Flex, Text } from "@artsy/palette"
+import { Artist } from "Apps/MyCollection/Routes/EditArtwork/Utils/artworkModel"
+
+export interface ArtistAvatarProps {
+  artist: Artist
+}
+
+export const ArtistAvatar: React.FC<ArtistAvatarProps> = ({
```
Comment (MounirDhahri) [inline src/Apps/MyCollection/Routes/EditArtwork/Components/ArtistAvatar.tsx:8]: **thought:** have you considered using https://palette.artsy.net/elements/entityheader/#gatsby-focus-wrapper

**Review by MounirDhahri — DISMISSED** [review]
LGTM. I left only one small comment

File: src/Apps/MyCollection/Routes/EditArtwork/Components/ArtistAvatar.tsx (hunk 2 above)
Comment (olerichter00) [reply src/Apps/MyCollection/Routes/EditArtwork/Components/ArtistAvatar.tsx]: Great idea! I'm using `EntityHeader` and `EntityHeaderArtistFragmentContainer` now 👍

**Review by MounirDhahri — APPROVED** [review]
Great! let's get this merged

**Review by MrSltun — APPROVED** [review]
Looks great! 🌟

**Pull request merged**